logviewer: tolerate pyqt4 + python2.7 better

mlyle opened the pull request:
Reported by Troy on IRC.

Siiiiigh.

Tested with full matrix py2 vs py3, pyqt4 vs pyqt5.

mlyle committed:
logviewer: tolerate pyqt4 + python2.7 better

Reported by Troy on IRC.

tracernz commented on python/dronin/logviewer/logviewer.py:
```diff
@@ -163,7 +163,9 @@ def handle_open(ignored=False, fname=None):
         global t
 
-        if not isinstance(fname, six.string_types):
+        print type(fname)
```
Did you mean this print to be here? This will break with Python3?

mlyle replied:
No, i didn't :P

mlyle commented:
Ugh needs more work, I must have failed the python3 test before.

mlyle committed:
python-api: logviewer: detect type of fname "right"

I'm sad that it comes down to this, but this is the only simple way I
could make dependable.

mlyle commented:
OK, this is what I'm goin' with. :P
